lib/settings/pages/general_settings_page.dart

```diff
@@ -641,6 +650,31 @@ class _GeneralSettingsPageState extends State<GeneralSettingsPage> with SingleTi
               ),
             ),
           ),
+          SliverToBoxAdapter(
+            child: Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 16.0),
+              child: ListOption(
+                description: l10n.imageCachingMode,
```

Comment:
Since #1086 has been merged in, we can add in `highlightKey` for search results here!

Reply:
You bet! Done!